openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/JavaGen.java

```diff
@@ -91,64 +92,72 @@ public static void main(String[] args)
 
 
 		JavaGen generator = new JavaGen();
-		generator.go(path, numArg);
-	 
+		generator.go(path, numArg, javaVersion);
+
 	}
-	
-	void go(String path, int numArg) {
+
+	void go(String path, int numArg, int  javaVersion) {
 		log("Creating source files");
-		makeJavas(path, numArg); //Make java source files
+		makeJavas(path, numArg,javaVersion); //Make java source files
```

Comment:
- need extra space
- is `javaVersion` used in makeJavas()?

Reply:
`javaVersion` seems to be not used in makeJavas. Will remove it there.